src/Connection/Aggregate/SentinelReplication.php

```diff
@@ -381,7 +381,7 @@ protected function querySentinelForSlaves(NodeConnectionInterface $sentinel, $se
         $slaves = array();
 
         $payload = $sentinel->executeCommand(
-            RawCommand::create('SENTINEL', 'slaves', $service)
+            RawCommand::create('SENTINEL', 'replicas', $service)
```

Comment:
This will break compatibility with older versions.

Reply:
Yeah sorry of course did not think of that.  Was fixated on that everything older then 7.0.x, 6.0.x and 6.2.x was EOL.

Reply:
I wish 🙃

Reply:
Misuse profile to signal 7.0.0 version from config time?

~~~
        if (version_compare($sentinel->getParameters()->profile, '7.0') >= 0) {
            $payload = $sentinel->executeCommand(
                RawCommand::create('SENTINEL', 'replicas', $service)
            );
        } else {
            $payload = $sentinel->executeCommand(
                RawCommand::create('SENTINEL', 'slaves', $service)
            );
        }
~~~

Reply:
> Yeah sorry of course did not think of that. Was fixated on that everything older then 7.0.x, 6.0.x and 6.2.x was EOL.

From a brief review of the Redis sources, SENTINEL REPLICAS is supported from Redis 5.0.